content/SCALE/SCALEUIReference/TopToolbar/Alerts/AlertSettingsScreen.md

```diff
@@ -8,123 +8,148 @@ tags:
 
-Option is **Plugin Update Available**.
+You can customize the alert setting for when plugin updates are available.
 {{< /expand >}}
+
 ### Network Alert Settings
-**Network** alert settings applies to network interfaces configured on your TrueNAS.
+**Network** alert settings apply to network interfaces configured on your TrueNAS.
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsNetwork](/images/SCALE/22.02/AlertSettingsNetwork.png "Network Alert Settings")
+![AlertSettingsNetwork](/images/SCALE/22.12/AlertSettingsNetwork.png "Network Alert Settings")
 
-Options cover LAGG interface ports status.
+You can customize alert settings for when ports are not active on the LAGG interface and when the LAGG interface has no active ports.
 {{< /expand >}}
+
 ### Reporting Alert Settings
-**Reporting** alert settings apply to collectd, reporting database and syslog processes on your TrueNAS.
+**Reporting** alert settings apply to collectd, database size threshold, and syslog processes on your TrueNAS.
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsReporting](/images/SCALE/22.02/AlertSettingsReporting.png "Reporting Alert Settings")
+![AlertSettingsReporting](/images/SCALE/22.12/AlertSettingsReporting.png "Reporting Alert Settings")
 
-Options cover collectd critical alerts and warnings, reporting database size threshold exceeded and syslog-ng is not running.
+You can customize alert settings for when collectd has critical alerts and warnings, the reporting database size exceeds the threshold, and syslog-ng is not running.
 {{< /expand >}}
+
 ### Sharing Alert Settings
-**Sharing** alert settings apply to iSCSI, NFS or SMB shares and connections configured on your TrueNAS.
+**Sharing** alert settings apply to iSCSI, NFS, or SMB shares and connections configured on your TrueNAS.
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsSharing](/images/SCALE/22.02/AlertSettingsSharing.png "Sharing Alert Settings")
+![AlertSettingsSharing](/images/SCALE/22.12/AlertSettingsSharing.png "Sharing Alert Settings")
 
-Options cover IP addresses bound to an iSCSI ports not found, NFS services not bound to specific IP addresses using 0.0.0.0, NFS share references hosts that cannot b resolved, NTLMv1 authentication attempted in the last 24hours, SMB1 connections to TrueNAS server performed in last 24 hous and share unavailable because it uses a locked dataset.
+You can customize alert settings for when a deprecated service is running, IP addresses bound to an iSCSI portal are not found, NFS services cannot bind to specific IP addresses using 0.0.0.0, and the system cannot resolve NFS share references hosts.
+
+You can also customize alerts for when NTLMv1 authentication has attempted in the last 24 hours, SMB1 connections to TrueNAS server have performed in the last 24 hours, and a share is unavailable because it uses a locked dataset.
```

Comment:
I'm thinking of a more active voice/present tense way to say this, maybe this? Or a variation of it?
```suggestion
You can also customize alerts for when NTLMv1 attempts authentication in the last 24 hours, SMB1 connections to TrueNAS server are performed in the last 24 hours, and a share is unavailable because it uses a locked dataset.
```